scripts/merge-and-build.sh

```diff
@@ -114,6 +139,8 @@ else
   fi
 fi # End check if we are master
 
+# TODO: Origin already vendors the web console. Should we stop updating it separately for OSE?
```

Comment:
With some extra validation in place, this can work. It is also a better idea to stick with rebasing (blacklisting) instead of resetting+cherry-picking (whitelisting) - should address Troy's concern.